src/app/performance-display/performance-display.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, OnInit, Output } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material';
+import { Inject } from '@angular/core';
+import { LanguageService } from '../services/language.service';
+import { ActivitySuccessObject } from './models/SuccessMessage';
+import { EventEmitter } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-performance-display',
+  templateUrl: './performance-display.component.html',
+  styleUrls: ['./performance-display.component.scss']
+})
+export class PerformanceDisplayComponent implements OnInit {
+  public SuccessObject: ActivitySuccessObject;
+  public language: any;
+  public activityName: string;
```

Comment:
can this be a model ?

Reply:
@shreyans29 Yes! ActivitySuccessObject I created for that purpose only. These activityName and other variables are for binding in the template.